css/edit.css

```diff
@@ -47,25 +47,28 @@
 }
 
 #edit-color gaia-color-picker {
-  flex: 1 0 70%;
+  flex: 1 1 70%;
```

Comment:
now the color picker can adjust when it has more or less available space (depending whether the autotheme palette is displayed or not)